source/global.h

```diff
@@ -10,16 +10,25 @@ WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
 PARTICULAR PURPOSE.  See the GNU General Public License for more details.
 */
 
-#ifndef PI_H_
-#define PI_H_
+#ifndef GLOBAL_H_
+#define GLOBAL_H_
 
 
 
+const int FRAME_RATE = 60;
```

Comment:
Probably every constant you are adding should actually be `constexpr` instead of just `const`